### DISCUSSION TITLE
Platform Level AI Assistant

### DISCUSSION THREAD

**Discussion by kkenna3** [twilio-labs/paste#4115] [Office hours] [answered]
### Background

We are exploring a platform level chat for the AI assistant currently used by support center and messaging in console. 
Our intention is to use the existing side panel used for the chat on Messaging Insights, but find a trigger that could be used across console in the future. We want to make sure it can be turned on/off on each page until the APIs integrated by the chat until it can support all products. 
For the first phase of the chat, the CTA would only appear on messaging and voice insights, and will be improved after public beta feedback. 

### Feedback

We'd like feedback on a potential direction (we are also user testing them) as well as advice on the current limitations of the side panel and if there are better options.
[ExplorationsI](https://www.figma.com/proto/0YUpwx8mVldwJxRZ15RcPp/2024-Q4-Voice-Insights-Wise-Owl?page-id=2044%3A56544&node-id=2044-68892&node-type=frame&viewport=-117%2C-193%2C0.25&t=pRqeVdkcBv5Jk4hB-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=2044%3A68892&show-proto-sidebar=1)

### Stage of design

Exploration

### Research

https://docs.google.com/document/d/1CatIr1ucoRZ6w61BQrROFA6VOHPQPJdzdkEUBur7rQU/edit?tab=t.0#heading=h.sbrsuapjzz62 

### Are you looking for visual design support?

- [X] Yes

**Comment by serifluous** [comment, marked as answer]
Thanks for coming to office hours today! You can find the recording in the #help-design-system channel in slack.

And let us know the results of your user preference test!

**Comment by kkenna3** [comment]
To follow up on this: what is the recommended interaction for multiple side panels?
Ie. Should the first side panel be closed or overlayed when opening a 2nd side panel

(let me know if I should create a new discussion!)

**Reply by serifluous** [reply]
The first panel should be closed since Side Panels will push content over and don't overlay.
